lib/whitehall_importer/integrity_checker.rb

```diff
@@ -2,6 +2,15 @@ module WhitehallImporter
   class IntegrityChecker
     attr_reader :edition
 
+    def self.time_similar?(proposed_time, publishing_api_time)
+      return true if proposed_time == publishing_api_time
+
+      Time.zone.rfc3339(proposed_time).between?(Time.zone.rfc3339(publishing_api_time) - 5,
+                                                Time.zone.rfc3339(publishing_api_time) + 5)
+    rescue ArgumentError
+      false
+    end
```

Comment:
I'm a bit confused why we're not testing this method in this commit? I notice the commit message says "Tests will be added in the next few commits." but there don't seem to be any explicit tests for this static public method.

I think they'd be quite useful as a way to document the 10 second window, which I think should also be documented in the commit message.

Reply:
I originally didn't test this method as all the scenarios were covered from the tests of the instance of the class which meant testing it seemed OTT, I have since removed a test as per your suggestion and added a full test block for the method.